package.json

```diff
@@ -1,5 +1,6 @@
 {
     "scripts": {
+        "build": "NODE_OPTIONS='--openssl-legacy-provider' npm run production",
```

Comment:
Can we please remove this? It's not related to the change of how permissions work.

Reply:
No problem. Would you like me to also fix the style checks that didn't pass? I'll have to learn how to configure PHP CS Fixer to conform to the repo style, but I can surely do this manually for now.

Reply:
Yes, please!